blog/2024-05-28-release-24-05.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Tractus-X 24.05 available now
+description: Release Tractus-X 24.05 announcement 
+slug: new-release-2405
+date: 2024-05-28T10:00
+hide_table_of_contents: false
+authors:
+  - name: Hanno Focken
+    title: Ecosystem & Kits
+    url: https://github.com/hfocken
+    image_url: https://github.com/hfocken.png
+    email: hanno.focken@catena-x.net
+---
+
+![Release Image](@site/static/img/Tractus-X_Release24.05.png)
+
+We are excited to announce the release of Tractus-X 24.05! This release builds the foundation for one of the most significant updates to our Catena-X ecosystem to date. Highlights include an update to the Multi-Opco solutions, the full implementation of the Industry Core KIT as well as the advancement of Self Sovereign Identity concepts, e.g. Multi-Wallet solutions. Conjointly, it makes several improvements to the user-friendliness of KITs based on feedback received to date, making Tractus-X overall more accessible and welcoming to newcomers and veterans alike. A big thank you to all contributors and committers for bringing this milestone to life and keep Tractus-X as one of the most significant OS projects in the automotive industry.
```

Comment:
AFAIK: Multi-Wallet Solutions is not enabled, yet. Please verify with @jjeroch .

Reply:
Done